types/@metamask/swappable-obj-proxy.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module '@metamask/swappable-obj-proxy' {
```

Comment:
Even though `swappable-obj-proxy` now contains type definitions, I missed that they didn't get included in the shipped version. This is because `src/index.d.ts` wasn't included in the set of `files` in `package.json`. So this copies the same file over (see https://github.com/MetaMask/swappable-obj-proxy/blob/main/src/index.d.ts). Not a big deal — when we convert this library to TypeScript, this will go away anyway, so this is short-lived.

Reply:
Now that https://github.com/MetaMask/swappable-obj-proxy/pull/27 exists, I'm going to mark this PR as blocked so that we don't have to do this.

Reply:
swappable-obj-proxy 2.1.0 has been released, so this PR is unblocked.